product_docs/docs/net_connector/8.0.5.1/18_api_reference.mdx

```diff
@@ -3,11 +3,10 @@ title: "API reference"
 
 ---
 
-<div id="api_reference" class="registered_link"></div>
-
 For information about using the API, see the [Npgsql documentation](http://www.npgsql.org/doc/api/Npgsql.html).
 
 Usage notes:
 
 -   When using the API, replace references to `Npgsql` with `EnterpriseDB.EDBClient`.
 -   When referring to classes, replace `Npgsql` with `EDB`. For example, use the `EDBBinaryExporter` class instead of the `NpgsqlBinaryExporter` class.
+-   See [Release Notes](./release_notes) and select the .NET EDB Connector version you are using to obtain the upstream Npgsql API version available with your installation.
```

Comment:
"upstream Npgsql API version available" is not clear, I propose "release notes link for the upstream Npgsql version".
Maybe I'm missing the point here ? Please advise

Reply:
What I intended to say is, to see which API Npgsql version was shipped with a specific EDB .NET version, you have to go to the EDB .NET release notes and see which Npgsql upstream merge was merged. Mainly because the availability of certain API features will vary from version to version. Let me try to rephrase and let me know what you think: 
```suggestion
-   To determine which Npgsql API version was included with a specific EDB .NET release, refer to the [EDB .NET release notes](./release_notes). The release notes specify the upstream Npgsql version that was merged. This is important because the availability of certain API features may vary between versions.
```

Reply:
OK for me, thanks for the rewording, LGTM.